### ISSUE TITLE
Create list of products to be used by gdalwarp for raster mosaic

### ISSUE THREAD

**Issue by mwhutchi** [SouthFACT/landsatfact-data#25]
Create database view for generating the "latest change" quad list - group by "quad_id", to get unique "change_id", sort by "product_date" and then by cloud cc% - view to supply list for GDALmerge - make sure to order list for freshest-clearest quad on top

**Comment by mwhutchi** [comment]
Was able to confirm that gdalwarp uses the sequential ordering for handling overlaps, so the file last in the list will be placed on top in the mosaic.

**Comment by mwhutchi** [comment]
Derek, William and I worked on this and have the first part of the view query working, which provides a list of tifs based on the scenes that were updated in the previous day:

```
 select location
 from vw_last_days_scenes as lds
 join view_tile_index as vti
 on lds.scene_id like '%'||substring(vti.quad_id, 2, 6)||'%'
```

The next part is ordering this list by quad cc %. With this, I am not sure how to get the quad cc for each tif since it is stored in separate columns in the landsat_metadata table. I wonder if instead, the cc_quad_xx field should be written to the extracted_imagery table in a single field where it can be joined via quad_id. Also, this field is one written by the geoprocessing script, which the other fields in landsat_metadata are not.
